styles.css

```diff
@@ -0,0 +1,72 @@
+@import url(//fonts.googleapis.com/css?family=Open+Sans:400,700,800,300);
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  min-height: 100vh;
+  justify-content: center;
+  align-items: center;
+  font-family: 'Inter', 'Open Sans', 'Helvetica Neue', Helvetica, sans-serif;
+}
+
+section {
+  width: 375px;
+  height: 6715px;
+  background: #fff;
+  border: 1px solid #ccc;
```

Comment:
- [ ] Please remove this border around your section. It is wrapping around the whole design. I have attached an image below. Please try to stick to the design guidelines as much as possible :100: 
![image](https://user-images.githubusercontent.com/102699958/212479734-c2c40b9e-4216-45ee-b91f-d0ec9abddfc3.png)